### PR TITLE
chore: migrate master to main by renaming master to main in all refer…

### PR DESCRIPTION
**Pull request by marianna-exelate**
**What**:

Changed the refs from the 'master' branch into 'main'

**Why**:

To be more compliment with updated web standards

**How**:

updated mainly the contribution guide and the validation file, kept 'master' branch available to be backward compatible as an intermediate phase

**Checklist**:

- [ x] Documentation updated
- [ x] Tests
- [ x] Ready to be merged
      
- [ x] Added myself to contributors table

### CODE REVIEW

**Comment by mpeyper** [comment]
Hi @marianna-exelate, thanks for this! It's been on my to-do list for a while.

Other than the references in the repo to `master`, do you know of any other steps that need to be taken (e.g. changing the default branch in the repo settings)?

File: .all-contributorsrc [hunk 1]
```diff
@@ -529,6 +529,15 @@
         "bug",
         "code"
       ]
+    },
```
Comment (MatanBobi) [inline .all-contributorsrc:532]: This is done automatically with a bot, you can remove this :)

**Comment by MatanBobi** [comment]
> Hi @marianna-exelate, thanks for this! It's been on my to-do list for a while.
> 
> Other than the references in the repo to `master`, do you know of any other steps that need to be taken (e.g. changing the default branch in the repo settings)?

I can help with that @mpeyper, we've done it in rtl already :) Let me merge this one and verify everything works, I'll do it later on today

**Comment by mpeyper** [comment]
Thanks @MatanBobi, Its late here so won't get to it myself for a while.

**Comment by MatanBobi** [comment]
Thanks @marianna-exelate! :)

**Comment by MatanBobi** [comment]
@all-contributors please add @marianna-exelate for infra :)

**Pull request merged**